internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -433,12 +433,16 @@ func (e *EnvStackConfig) vpcConfig() (template.VPCConfig, error) {
 	if err != nil {
 		return template.VPCConfig{}, err
 	}
+	vpcFlowLogsArgs, err := convertFlowLogsConfig(e.in.Mft)
```

Comment:
nit: I really like this guide for naming variables: https://a-nickels-worth.blogspot.com/2016/04/a-guide-to-naming-variables.html#orgheadline12. Take a look at the entire article if you have time.

Since this variable is declared close to where it's used I think it's ok if we have a shorter name:
```suggestion
	flowLogs, err := convertFlowLogsConfig(e.in.Mft)
```